nf_core/__main__.py

```diff
@@ -372,7 +372,7 @@ def create_params_file(pipeline, revision, output, force, show_hidden):
     is_flag=True,
     default=False,
     hidden=True,
-    help="Download for Seqera Platform. DEPRECATED: Please use --platform instead.",
+    help="Download for Seqera Platform. DEPRECATED: Please use `--platform` instead.",
 )
 @click.option(
     "-t",
```

Comment:
@MatthiasZepper doesn't like this and I agree that it's a bit odd. Let's just remove the short form here.
```suggestion
```